### ISSUE TITLE
Datasets mismatch

### ISSUE THREAD

**Issue by nathalia-kim** [hbkunn/nu_gan#4]
Hello, 

Thank you for making this code available! 

There's a mismatch between the Datasets described in the paper and the data provided on this page. 

**From the paper description:** 
* Dataset A has 11 negative images
  *  includes 1995 cell-level images
* Dataset B has 29 positive images 
* Dataset C = Dataset A + Dataset B
* Dataset D has 72 positive and 132 negative images

**Data provided here:** 
* dataset_A folders:
  * original > negative_images: contains 11 images (corresponding to Dataset A from the paper)
  * original > positive_images: contains 29 images (corresponding to Dataset B from the paper)
  * cell_level_label: contains 4 files (X_train.npy, X_test.npy, y_train.npy, y_test.npy)
     * len(X_train) = 1631
     * len(X_test) = 364
     * 1631 + 364 = 1995
     *  So these 4 files most probably correspond to the cell-level images from Dataset A and their labels 

* dataset_B folders:
   * original > negative_images: contains 132 images (corresponding to Dataset D from the paper)
   * original > positive_images: contains 72 images (corresponding to Dataset D from the paper)
   * cell_level_label: contains exactly the same 4 files as dataset_A > cell_level_label, therefore correspond to the cell-level images from Dataset A and their labels 

In summary, the original images from all datasets in the paper were provided. However, **only the cell-level images and labels for Dataset A are available**. Would it be possible to provide those for the other datasets as well? 

Thank you

**Comment by bohu615** [comment]
---

**Comment by nathalia-kim** [comment]
Sorry I didn't understand your answer, what do you mean by "@*** @ *** . *** *** @ *** . ***>"?

**Comment by bohu615** [comment]
Sorry about the formatting issue. That meant to be a reply to the email. The *** are the email address filtered by github.

Please email the co-author (Ye Tang) in the paper for full dataset access. 

His email can be found in the paper. I currently don't have the full images on my computer.

Best,
Bo

**Comment by nathalia-kim** [comment]
Oh okay, no worries! Thank you for the reply, I will contact him about this issue!
